shared/src/business/entities/trialSessions/SpeciailTrialSessions.ts

```diff
@@ -0,0 +1,14 @@
+export interface SpecialTrialSession {
```

Comment:
I think it is probably better to put these into the interactor file itself. I also wonder if we can rename `SpecialTrialSession`, which sounds like it is the full SpecialTrialSession type. (BTW: I have been told we use `type` more often than `interface`. I'm not sure how important that is, but I figured I would mention it.)

Reply:
I attempted this but because I'm using these types in both client side and server side code, it was easier to put them here without getting errors like 'module not found' etc.